### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by kawaMANMI** [codewars/codewars.com#2985] [Q&A]
Dear Codewars Team,

 

I have been away from Codewars for over two months. Today, upon checking my account, I was surprised to discover that my Kata marks had dropped from almost 1000 to 428. I haven't engaged with the platform in the last two months, but prior to that, I was regularly practicing and solving katas. Could you please clarify what's happening? I attempted to research the term "Account restricted for violation of terms of service," and it appears to be associated with plagiarizing other users' code. However, I don't think that applies to me because my account was in good standing for an extended period. Additionally, I have already archived the solutions for most of my katas. If there are any issues, please inform me, and I can provide additional information.

 

 

My user name is: KawaManmi

and name is Kawa Manmi

 

https://www.codewars.com/users/KawaManmi/stats

 

 

Kind regards,

Kawa

https://github.com/kawaMANMI

https://www.linkedin.com/in/kawa-manmi-5244ab53/

**Comment by Blind4Basics** [comment]
discussed there: https://github.com/codewars/codewars.com/issues/2986

closing here.
